### ISSUE TITLE
Netlify CMS starter

### ISSUE THREAD

**Issue by Larsklopstra** [gridsome/gridsome#304]
## Summary

The Netlify CMS boilerplate code

### Motivation

It's too much work to create the same project over and over again, that's why I'd like to create a Netlify CMS starter

**Comment by tomtev** [comment]
Yes, we would have that :) Could you make a repo named "gridsome-starter-netlify-cms". We can move it to the gridsome organization when it's done.

**Comment by Larsklopstra** [comment]
Check

**Comment by Larsklopstra** [comment]
@tomtev It's done but there are some issues which have to be resolved in the future, here's the repo as of now: [gridsome-starter-netlify-cms](https://github.com/Larsklopstra/gridsome-starter-netlify-cms).

There are some issues to be aware of: you can't use progressive images due a forced leading `/` which is done on Netlify's end, the temporary work around I've added is placing the images into `static/uploads` so they will remain visible on the website, I'd still advice to lazy load them through `IntersectionObservers` to performant, but that's the choice of the end user.

Let me know what you think 🤷‍♂️

**Comment by Larsklopstra** [comment]
@tomtev [Site preview](https://kind-lalande-7c5833.netlify.com/), ready for review

**Comment by suits-at** [comment]
I didn't see this issue and build something similar [(netlifycms-gridsome)](https://github.com/suits-at/netlifycms-gridsome) a few days ago, on top of the [gridsome-starter-blog](https://github.com/gridsome/gridsome-starter-blog). I link it here, because it might be useful as a starting point for people who want to use gridsome-starter-blog combined with netlify-cms. The repository itself seems to be quite similar to what @Larsklopstra has done, main difference might be that I inserted netlify-cms directly without using the [gridsome-plugin-netlify-cms](https://github.com/gethypervisual/gridsome-plugin-netlify-cms).

**Comment by suits-at** [comment]
Meanwhile my repo [netlifycms-gridsome](https://github.com/suits-at/netlifycms-gridsome) also got merged into [netlifys JAMstack templates](https://templates.netlify.com/template/netlifycms-gridsome/). So I think it should work just fine. :)

**Comment by robaxelsen** [comment]
@tomtev have you had a chance to have a look at the starter by @Larsklopstra?

@suits-at thanks for working on this in parallel as well! Good work. Perhaps we can add a link to your started in the docs.

**Comment by absentees** [comment]
@suits-at thanks for your repo was very helpful. i found that how you have the images setup does not trigger gridsome to process the image with `g-image`. It's got to do with the path of the image which has it sort of linked to #292 

There is a gridsome plugin that renames the paths in your markdown to correctly reference the images and process them. I'll try to send you a pull request.

**Comment by papeloto** [comment]
At some point I created the project following the guide steps:
https://github.com/papeloto/gridsome-netlify-cms

Maybe you can fork or I can help somehow? :smile:
